lambda/src/main/resources/db/migration/V5__add_checksum_to_file_property.sql

```diff
@@ -0,0 +1 @@
+insert into "FileProperty" ("PropertyId" , "Name", "Description", "Shortname") VALUES (uuid_generate_v4(), 'ServerSideChecksum', 'The checksum calculated after upload', 'Checksum');
```

Comment:
Very minor but should capitalize "INSERT INTO" just for consistencies sake

Reply:
Fair point :smile: I've changed it.